package.json

```diff
@@ -15,7 +15,7 @@
         "test:unit": "vue-cli-service test:unit",
         "test:e2e": "vue-cli-service test:e2e",
         "lint": "vue-cli-service lint",
-        "test": "vue-cli-service test:unit --coverage",
+        "test": "vue-cli-service test:unit --coverage && vue-cli-service test:e2e",
```

Comment:
I added the --coverage option when I was writing the first updated version, together with the tests. We could remove it as well, but I think it can be useful for Outreachy/ GSoC candidates to see if their tests cover all lines or not.

Reply:
Agreed, let's leave it in